roles/core/node/tasks/install_repository.yml

```diff
@@ -13,6 +13,19 @@
    gpgcheck: "no"
   when: scale_version < '5.0.4.0'
 
+# Copy and import gpg key on RHEL and SLES if gpfs version >= 5.0.5.0
+- block:
+    - name: check | Copy key
+      get_url:
+        url: "{{ scale_gpgKey_repository_src }}"
+        dest: "{{ scale_gpgKey_dest }}"
+
+    - rpm_key:
+        state: present
+        key: "{{ scale_gpgKey_dest }}SpectrumScale_public_key.pgp"
```

Comment:
Might want to add an extra slash `/` here to be on the safe side:
```
"{{ scale_gpgKey_dest }}/SpectrumScale_public_key.pgp"
```

Reply:
Fixed.